examples/scheduled-tasks/architect.yml

```diff
@@ -11,10 +11,7 @@ tasks:
     command:
       - sh
       - -c
```

Comment:
Why the ingress url? Is this because the consul sidecar for the task wasn't working as expected?

Reply:
Well there currently isn't a sidecar for k8s tasks, so the external endpoint will have to do until that work is completed